addons/account/models/chart_template.py

```diff
@@ -340,7 +340,6 @@ def tax_template_changed(tax, template):
                 or len(template_line_ids) not in (0, len(tax.repartition_line_ids))
             )
 
-        existing_current_year_earnings_account = self.env['account.account'].search([('company_ids', '=', company.id),('account_type', '=', 'equity_unaffected')], limit=1)
```

Comment:
Note that the call to `get_unaffected_earnings_account` in `l10n_nl` needs to take into account all the `equity_unaffected` accounts